internal/controller/templatechain_controller_test.go

```diff
@@ -231,9 +231,6 @@ var _ = Describe("Template Chain Controller", func() {
 
 				Eventually(k8sClient.Get, 1*time.Minute, 5*time.Second).WithArguments(ctx, chain, clusterTemplateChainResource).Should(HaveOccurred())
 			}
-			for _, template := range []*hmcmirantiscomv1alpha1.ClusterTemplate{ctTemplates["test"], ctTemplates["ct0"], ctTemplates["ct2"]} {
```

Comment:
Removing the templates' cleanup since the webhook now denies these requests. Templates have owner reference set and should be automatically deleted by the garbage collector. The owner reference existence is checked below.

Reply:
Redundant since the admission validation webhook was disabled in tests.